pkg/services/dashboards/service/dashboard_service.go

```diff
@@ -53,13 +52,7 @@ import (
 )
 
 var (
-	provisionerPermissions = []accesscontrol.Permission{
-		{Action: dashboards.ActionFoldersCreate, Scope: dashboards.ScopeFoldersAll},
-		{Action: dashboards.ActionFoldersWrite, Scope: dashboards.ScopeFoldersAll},
-		{Action: dashboards.ActionDashboardsCreate, Scope: dashboards.ScopeFoldersAll},
-		{Action: dashboards.ActionDashboardsWrite, Scope: dashboards.ScopeFoldersAll},
```

Comment:
This one seems to be missing from the Grafana Identity. Is it because it's not actually needed?

Reply:
Good catch, lets add it just in case